content/howto/geodesic/authenticate-with-aws-vault.md

```diff
@@ -2,7 +2,6 @@
 title: "Authenticate with AWS inside Geodesic using `aws-vault`"
 description: "Learn how to authenticate within Geodesic using AWS IAM Credentials and `aws-vault`."
 weight: 1
-draft: true
```

Comment:
@osterman note that this removes this document from draft since I linked out to it. I don't believe it was intentionally supposed to be in draft when it was originally merged, so that is my bad.